source/common/modules/markdown-editor/plugins/highlight-whitespace.ts

```diff
@@ -49,16 +52,30 @@ class PilcrowWidget extends WidgetType {
   }
 }
 
-const pilcrowDeco = Decoration.widget({ widget: new PilcrowWidget(), side: 1 })
+// The widget should appear last, so we set it to the highest side
+const pilcrowDeco = Decoration.widget({ widget: new PilcrowWidget(), side: 10000 })
 
 function showLineEndings (view: EditorView): DecorationSet {
   const builder = new RangeSetBuilder<Decoration>()
   for (const { from, to } of view.visibleRanges) {
-    for (let pos = from; pos <= to;) {
-      const line = view.state.doc.lineAt(pos)
-      builder.add(line.to, line.to, pilcrowDeco)
-      pos = line.to + 1
-    }
+    syntaxTree(view.state).iterate({
+      from,
+      to,
+      enter (node) {
+
+        // Don't show pilcrows in code text
+        if (node.name === 'CodeText') { return false }
```

Comment:
Interesting decision. Is this how other apps also work..? Or would we be the trend setters here? I mean I can see the logic, just for clarification.